pkg/deploy/config.go

```diff
@@ -147,6 +150,13 @@ func (conf *RPConfig) validate() error {
 	v := reflect.ValueOf(*configuration)
 	missingFields := []string{}
 
+	if conf.Configuration.SSHPublicKey == nil {
+		key, _ := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Should we handle the error here?

Reply:
absolutely. if `err!=nil` you have a nil pointer and will probably panic on the next line.

Reply:
Thanks, both. Added error handling. Please review.